### DISCUSSION TITLE
Rotate Angle for line Chart is not available

### DISCUSSION THREAD

**Discussion by Salman3sh** [imaNNeo/fl_chart#1156] [Q&A]
Hi,

I am trying to rotate the bottom-line data to be vertical for the line chart( I have datetime data and it is long) but it seems that rotateAngle Property is not part of sideTitles anymore. How could I rotate the data at
 the bottom?
![2022-09-29 12_44_40-](https://user-images.githubusercontent.com/69638951/193011678-3aca723d-3569-40a6-8368-eda56821b8e0.png)

**Comment by imaNNeo** [comment]
We had some breaking changes in the [0.50.0](https://github.com/imaNNeo/fl_chart/releases/tag/0.50.0) release. So you can read the migration guide [here](https://github.com/imaNNeo/fl_chart/blob/master/repo_files/documentations/migration_guides/0.50.0/MIGRATION_00_50_00.md).
